client/post-editor/media-modal/detail/detail-preview-videopress.jsx

```diff
@@ -58,38 +74,93 @@ class EditorMediaModalDetailPreviewVideoPress extends Component {
 	}
 
 	onScriptLoaded = ( error ) => {
+		const {
+			isPlaying,
+			item,
+			onScriptLoadError,
+		} = this.props;
+
 		if ( error ) {
-			log( `Script${ error.src } failed to load.` );
+			log( `Script${ get( error, 'src', '' ) } failed to load.` );
+			onScriptLoadError();
+
 			return;
 		}
 
 		const {
-			isPlaying,
-			item,
-		} = this.props;
+			height = 480,
+			videopress_guid,
+			width = 854,
+		} = pick( item, [ 'videopress_guid', 'height', 'width' ] );
```

Comment:
How bizarre! I can honestly say I have no idea why I did that. Regardless, it's fixed by https://github.com/Automattic/wp-calypso/pull/11653/commits/f49fbb8d75fcd3b2ab0fd5d8a6ce42eb8e73adab.